src/app/containers/ConsentBanner/Canonical/logic/README.md

```diff
@@ -0,0 +1,53 @@
+# Consent Banner Utilities
+
```

Comment:
Super useful README 👍 thanks Drew. Could you add to it to explicitly describe which cookies we're dealing with? e.g. I get there's 'privacy' and there's 'cookie' but would have assumed 2 cookies rather than 3.

Also pondering if it's worth adding features/scenarios from #1399 into the README to make it clear what the values should be, etc, but maybe that's opening a can of worms.

Reply:
Also worth documenting that the component sets both the explicit cookie, _and_ the cookie oven (and that the oven sets the same cookie across multiple TLDs).

Reply:
Ive added a small bit of text, happy to make an issue to document this better though, as to not block this PR